chapters/en/slides/module2/module2_05.md

```diff
@@ -0,0 +1,336 @@
+---
+title: 'module1_05'
+type: slides
+---
+
+# Working with Files and Directories
+
+Notes:
+We now know how to explore files and directories, but how do we create them? 
+That's what we will find out in this chapter.
+
+---
+
+## Creating a directory
+
+```sh
+cd ~/Documents
+
+# MaKe Directory
+mkdir notes
+```
+
+**Naming guidelines**
+
+1. Don't use spaces
+2. Don't begin the name with a `-` (hyphen)
+3. Stick to digits and letters (preferably lower case)
+
+Notes:
+Let's go back to the `~/Documents` directory and create a subfolder called `notes`.
+For this we first use the `~` home  directory shortcut,
+and then the `mkdir` command.
+
+We will talk more about useful naming conventions in module 6.
+For now it is enough if you remember these three guidelines.
+
+1. Don't use spaces
+2. Don't begin the name with a `-` (hyphen)
+3. Stick to digits and letters (preferably lower case)
+
+---
+
+## Creating a file
+
+```sh
+ls -F notes
+```
+
+<br>
+
+```sh
+cd notes
+code my-first-note.md
+```
+
+Notes:
+Since we just created the `notes` directory,
+ls doesn't display anything when we ask for a listing of its contents.
+
+Let’s change our working directory to `notes`,
+then use the VS Code editor to create a file called my-first-note.md.
+The command to invoke VS Code from the command line is `code`,
```

Comment:
Should be good to add something as... "You can try it now!" So they start interacting with the material